CMakeLists.txt

```diff
@@ -56,6 +56,7 @@ OPTION(WANT_MP3LAME	"Include MP3/Lame support" ON)
 OPTION(WANT_OGGVORBIS	"Include OGG/Vorbis support" ON)
 OPTION(WANT_PULSEAUDIO	"Include PulseAudio support" ON)
 OPTION(WANT_PORTAUDIO	"Include PortAudio support" ON)
+OPTION(WANT_SNDIO	"Include sndio support" OFF)
```

Comment:
Why did you turn off Sndio by default? I think you can turn this on and disable on Windows.

Reply:
> Why did you turn off Sndio by default? I think you can turn this on and disable on Windows.

Done.

I was trying to be conservative and keep the behavior on non-OpenBSD
systems the same as before.